src/HasMedia/HasMediaTrait.php

```diff
@@ -69,6 +69,28 @@ public function addMediaFromRequest(string $key)
     }
 
     /**
+     * Add multiple files from a request by keys.
+     *
+     * @param string[] $keys
+     *
+     * @return \Spatie\MediaLibrary\FileAdder\FileAdder[]
+     */
+    public function addMultipleMediaFromRequest(array $keys)
```

Comment:
This was done mainly to have a clear difference between `addMediaFromRequest` returning a `FileAdder` instance and `addMultipleMediaFromRequest` returning an array of `FileAdder`s. 

Personally I'd prefer a variadic method as well but I'm not sure if that's considered a breaking change?